samples/androidNativeActivity/gradle.properties

```diff
@@ -5,7 +5,7 @@ org.gradle.parallel=true
 org.gradle.workers.max=4
 
 # Pin Kotlin version:
-kotlin_version=1.3.30-eap-113
+kotlin_version=1.3.30
```

Comment:
Maybe we shall add easily discoverable marker, like CHANGE_KOTLIN_VERSION_HERE in comment, so that it's easy not to miss all such cases?

Reply:
I've added the `CHANGE_VERSION_WITH_RELEASE` marker and mentioned it in the release doc.

Reply:
Marker is better than nothing. :+1:

But still it requires editing multiple files. I think we could elaborate a better approach when only one file needs to be edited. Added this to https://jetbrains.quip.com/RO9VA7uCbqjF (point #4)